aspnetcore/host-and-deploy/linux-apache.md

```diff
@@ -320,6 +304,52 @@ rich rules:
 
 ### HTTPS configuration
 
+**Configure the app for secure (HTTPS) local connections**
+
+The [dotnet run](/dotnet/core/tools/dotnet-run) command uses the app's *Properties/launchSettings.json* file, which configures the app to listen on the URLs provided by the `applicationUrl` property (for example, `https://localhost:5001;http://localhost:5000`).
+
+Configure the app to use a certificate in development for the `dotnet run` command or development environment (F5 or Ctrl+F5 in Visual Studio Code):
+
+::: moniker range=">= aspnetcore-3.0"
+
+```csharp
+Host.CreateDefaultBuilder(args)
```

Comment:
Rather than giving a hardcoded example we should link to Kestrel's cert config docs. There are lots of ways to provide a certificate.
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-2.2#endpoint-configuration.

Reply:
Ok, but the content/example over there isn't explicit. When I do the commit to link these two spots over, I'll add a line over there to an existing 2.x example and possibly go ahead and add the 3.0 example. (Kestrel topic hasn't been updated for 3.0 yet).